### PR TITLE
[website] add missing authorization + PR previews

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Fixes the website white screen bug
- Adds website authorization environmental variable
- Requires a PR to be merged for merge dependant jobs to run.
- Adds website PR previews

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Checkout the [PR preview link](https://website-pr-previews-dot-airqo-frontend.appspot.com/) commented by the GitHub bot on the PR.
* If you can access the website, then all is good.
*(Not required, but if you want to run it locally, follow the next instruction)*
* Build and run the application in a Docker container as described [here](https://github.com/airqo-platform/AirQo-frontend/tree/fix-websiteWhiteScreen/website#docker-1) in the README. 

#### What are the relevant tickets?
- [WEB-330](https://airqoteam.atlassian.net/browse/WEB-330?atlOrigin=eyJpIjoiMjg0MGM4NTU0YmRlNDJlZWFkOTZkMjkzYTUxMmEzYmQiLCJwIjoiaiJ9)

#### Screenshots (optional)
- image

### CODE REVIEW

File: website/Dockerfile [hunk 1]
```diff
@@ -8,12 +8,14 @@ ARG REACT_NETMANAGER_BASE_URL
...
 ARG REACT_APP_BASE_NEWSLETTER_URL
 ARG REACT_APP_WEBSITE_BASE_URL
+ARG REACT_APP_AUTHORIZATION_TOKEN
 
 ENV REACT_WEB_STATIC_HOST=$REACT_WEB_STATIC_HOST
 ENV REACT_NETMANAGER_BASE_URL=$REACT_NETMANAGER_BASE_URL
 ENV REACT_APP_BASE_AIRQLOUDS_URL=$REACT_APP_BASE_AIRQLOUDS_URL
 ENV REACT_APP_BASE_NEWSLETTER_URL=$REACT_APP_BASE_NEWSLETTER_URL
 ENV REACT_APP_WEBSITE_BASE_URL=$REACT_APP_WEBSITE_BASE_URL
+ENV REACT_APP_AUTHORIZATION_TOKEN=$REACT_APP_AUTHORIZATION_TOKEN
```
Comment (123MwanjeMike) [inline website/Dockerfile:18]: Add missing build argument

File: .github/workflows/deploy-frontends-to-production.yml [hunk 2]
```diff
@@ -208,14 +209,9 @@ jobs:
           echo "  GS_BUCKET_NAME: ${{ secrets.WEBSITE_PROD_GS_BUCKET_NAME }}" >> .env.yaml
           echo "  REACT_WEB_STATIC_HOST: ${{ secrets.WEBSITE_PROD_REACT_WEB_STATIC_HOST }}" >> .env.yaml
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:210]: The only REACT_ variable used by the API

File: .github/workflows/deploy-frontends-to-production.yml [hunk 3]
```diff
@@ -208,14 +209,9 @@ jobs:
...
           echo "  SECRET_KEY: ${{ secrets.WEBSITE_PROD_SECRET_KEY }}" >> .env.yaml
-          echo "  REACT_NETMANAGER_BASE_URL: ${{ secrets.WEBSITE_PROD_REACT_NETMANAGER_BASE_URL }}" >> .env.yaml
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:211]: Remove API unused ENV vars

**Review by Codebmk — APPROVED** [review]
LGTM

File: website/README.md [hunk 4]
```diff
@@ -306,6 +317,7 @@ Build the application docker image with the command below. Make sure that your `
         --build-arg REACT_APP_BASE_AIRQLOUDS_URL=<<enter REACT_APP_BASE_AIRQLOUDS_URL value>> \
         --build-arg REACT_APP_BASE_NEWSLETTER_URL=<<enter REACT_APP_BASE_NEWSLETTER_URL value>> \
         --build-arg REACT_APP_WEBSITE_BASE_URL=<<enter REACT_APP_WEBSITE_BASE_URL value>> \
+        --build-arg REACT_APP_AUTHORIZATION_TOKEN=<<enter REACT_APP_AUTHORIZATION_TOKEN value>> \
```
Comment (123MwanjeMike) [inline website/README.md:320]: Add token ENV var to readme

**Review by FaithDaka — APPROVED** [review]

**Pull request merged**